data/data/agent/files/usr/local/bin/get-interactive-config.sh

```diff
@@ -0,0 +1,96 @@
+#!/bin/bash
+
+function connectivity_check() {
+	declare target
+
+	target="$1"
+
+	(>&2 echo "Pinging ${target} to check the validity of the current network configuration")
+	if ping -c 4 "$NODE_ZERO_IP"; then
+		(>&2 echo "Successfully pinged ${target}.")
+		return 0
+	else
+		(>&2 echo "Failed to ping ${target}.")
+		return 1
+	fi
+}
+
+function print_addressing_info() {
+	declare with_prefix
+
+	with_prefix="${1:-true}"
+	echo "Currently configured addresses"
+	if $with_prefix; then
+		ip -j a show  | jq -r '. | map({name: .ifname, addresses: [.addr_info[] | .local + "/" + (.prefixlen | tostring)]}) | .[] | select(.addresses | length > 0) | select( .name != "lo") | .name + ": " + (.addresses | join(" "))'
+	else
+		ip -j a show  | jq -r '. | map({name: .ifname, addresses: [.addr_info[] | .local]}) | .[] | select(.addresses | length > 0) | select( .name != "lo") | .name + ": " + (.addresses | join(" "))'
+	fi
+}
+
```

Comment:
I'm assuming we need to add something here that checks if node0 is defined in the ISO. If it is defined and connectivity check passes, then the following loops should be skipped after a predetermined amount timeout (according to https://issues.redhat.com/browse/AGENT-455)